lib/Echidna/Exec.hs

```diff
@@ -159,14 +158,16 @@ pointCoverage l = do
                     (fromMaybe (error "no contract information on coverage") $ h v)
                     mempty
   where
-    h v = getBytecodeMetadata <$>
-            v ^? env . contracts . at (v ^. state . contract) . _Just . bytecode
+    h v = do
+      buffer <- v ^? env . contracts . at (v ^. state . contract) . _Just . bytecode
+      bc <- viewBuffer buffer
+      pure $ getBytecodeMetadata bc
 
 traceCoverage :: (MonadState x m, Has VM x, Has [Op] x) => m ()
 traceCoverage = do
   v <- use hasLens
-  let c = v ^. state . code
-  hasLens <>= [readOp (BS.index c $ v ^. state . pc) c]
+  let ConcreteBuffer c = v ^. state . code
```

Comment:
@ggrieco-tob @incertia this is an irrefutable pattern, I didn't know what to do with the symbolic buffer here. Let me know if you have a better idea

Reply:
this function is not used anywhere actually

Reply:
Let's remove it.